.circleci/config.yml

```diff
@@ -1,117 +1,89 @@
 version: 2.1
 
-# The section below specifies YAML anchors to create reusable configuration
-# To Learn more about YAML anchors: https://blog.daemonl.com/2016/02/yaml.html
-
-setup_workspace: &setup_workspace
-  working_directory: ~/artsy/reaction
-  docker:
-    - image: circleci/node:8
-
-# https://circleci.com/docs/2.0/caching/#basic-example-of-dependency-caching
-save_cache: &save_cache
-  save_cache:
-    key: yarn-deps-{{ checksum "yarn.lock" }}
-    paths:
-      - ./node_modules
-
-# https://circleci.com/docs/2.0/caching/#restoring-cache
-restore_cache: &restore_cache
-  restore_cache:
-    # If there isn't a match to the first key, it'll do a partial match of the
-    # second. That means after the first cache save there will always be a cache
-    # hit, but it might be an older version of the cache
-    keys:
-      - yarn-deps-{{ checksum "yarn.lock" }}
-      - yarn-deps-
-
-# By default when yarn runs it does not check the filesystem to ensure the
-# packages it expects to be installed are actually installed. Using the
-# --check-files flag ensures that any packages or files missing or out of date
-# on the file system (i.e. those which might be restored from cache) match what
-# the yarn.lock file specifies
-verify_dependencies: &verify_dependencies
-  run: yarn --check-files
-
-filter_master: &filter_master
-  filters:
-    branches:
-      only:
-        - master
+orbs:
+  node: artsy/node@volatile
+  yarn: artsy/yarn@volatile
+  queue: eddiewebb/queue@volatile
```

Comment:
Any chance a comment could link to the two artsy references?

Reply:
Do you mean links to the repo?

Reply:
Although docs will use `volatile` as easy way to reference the latest, I would suggest pointing to specific versions for each orb for deterministic behavior over time.

orb cli or online [registry](https://circleci.com/orbs/registry/) are easy easy to check version and source

Reply:
Actually, easiest way is to pick a job that passed, and look at the top of the config file, https://circleci.com/gh/artsy/reaction/15151#config/containers/0 and you'll see

```
# Orb 'artsy/node@volatile' resolved to 'artsy/node@0.0.1'
# Orb 'artsy/yarn@volatile' resolved to 'artsy/yarn@0.0.1'
# Orb 'eddiewebb/queue@volatile' resolved to 'eddiewebb/queue@1.0.107'
```

Reply:
Cool, thanks. I'll update it.